index.html

```diff
@@ -78,6 +78,17 @@ <h2>Add New Recipe</h2>
 
     <div id="recipe-container"></div>
 
+    <div id="image-container"></div>
+
+    <div id="recipe-container"></div>
```

Comment:
There is already a `div` with this ID on line 79. Be careful to keep your IDs unique.